Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM node:12-slim
-
+RUN apt update && apt install -y git
```

Comment:
maybe we can have a heavier node image that includes git

Reply:
I am updating this in another PR